travel-event-host-app/src/lib/mongodb.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+
+const { MONGODB_URI } = process.env;
+
+if (!MONGODB_URI) {
+  throw new Error("Invalid enviroment variable: MONGODB_URI");
+}
+
+export const connectMongoDB = async () => {
+  try {
+    const { connection } = await mongoose.connect(MONGODB_URI);
+
+    if (connection.readyState === 1) {
+      return Promise.resolve(true);
+    }
+  } catch (error) {
+    console.log("Error connecting to MongoDB: ", error);
+
+    return Promise.reject(error);
```

Comment:
If we want to rethrow the error here, we can `throw new Error()`?

Reply:
yes